src/App.jsx

```diff
@@ -19,11 +21,12 @@ function App() {
         <Routes>
           <Route path="/" element={<Main />} />
           <Route path="/birthDay" element={<BirthDay />} />
-          <Route path="/attendance" element={<Attendance />} />
           <Route path="/news" element={<News />} />
+          <Route path="/attendance" element={<Attendance />} />
+          <Route path="/newsAdd" element={<NewsAdd />} />
```

Comment:
```suggestion
          <Route path="/newsAdd" element={<NewsAdd />} />
```
뉴스 수정/추가는 새로운 페이지로 가기보다 모달이나 드로어형태로 되면 좋을 듯

Reply:
추가 / 수정 화면을 드로어에 띄울거면 이쪽 라우팅은 필요없는거아닌가?